qiita_pet/templates/sitebase.html

```diff
@@ -49,10 +49,23 @@
         objbrowserName = M[0];
         objBrMajorVersion = M[1];
 
-        if ((objbrowserName == "Chrome"  && objBrMajorVersion < {{allowed_min_browser_versions['chrome']}}) ||
-            (objbrowserName == "Firefox" && objBrMajorVersion < {{allowed_min_browser_versions['firefox']}}) ||
-            (objbrowserName == "Safari"  && objBrMajorVersion < {{allowed_min_browser_versions['safari']}}) ||
-            ((objbrowserName == "IE" || objbrowserName == "MSIE") && objBrMajorVersion < {{allowed_min_browser_versions['IE']}})) {
+        if ((objbrowserName == "Chrome"  && objBrMajorVersion < {{allowed_min_browser_versions['chrome']}})
+            {% if 'firefox' in allowed_min_browser_versions %}
+              || (objbrowserName == "Firefox" && objBrMajorVersion < {{allowed_min_browser_versions['firefox']}})
```

Comment:
Won't this fail with a KeyError, since 'firefox' is no longer in this dictionary?